app/helpers/policy_dashboard_helper.rb

```diff
@@ -8,16 +8,16 @@
 # along with this software; if not, see http://www.gnu.org/licenses/gpl.txt
 #
 
-module ScaptimonyPolicyDashboardHelper
-  Colors = {
-    :compliant_hosts => '#89A54E',
-    :incompliant_hosts => '#AA4643',
-    :inconclusive_hosts => '#DB843D',
+module PolicyDashboardHelper
+  COLORS = {
+    :compliant_hosts => ArfReportDashboardHelper::COLORS[:passed],
+    :incompliant_hosts => ArfReportDashboardHelper::COLORS[:failed],
+    :inconclusive_hosts => ArfReportDashboardHelper::COLORS[:othered],
     :report_missing => '#92A8CD',
   }
```

Comment:
Excellent, however, this is not applied later in [line 37 ](https://github.com/xprazak2/foreman_openscap/blob/openscap-refactoring-backup-2/app/helpers/policy_dashboard_helper.rb#L37) and it raises `uninitialized constant PolicyDashboardHelper::Colors` error when visiting the policy's dashboard page.